scripts/setup.sh

```diff
@@ -14,66 +14,61 @@
+  echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }'
 }
 
 echo -e "\033[1;34mDynatrace function for Google Cloud Platform monitoring"
 echo -e "\033[0;37m"
 
-if ! command -v yq &> /dev/null
-then
+if ! command -v yq &> /dev/null; then
 
-    echo -e "\e[91mERROR: \e[37m yq and jq is required to install Dynatrace function. Please refer to following links for installation instructions:"
-    echo -e
-    echo -e "YQ: https://github.com/mikefarah/yq"
-    if ! command -v jq &> /dev/null
-    then
+    err 'yq and jq is required to install Dynatrace function. Please refer to following links for installation instructions:
+    YQ: https://github.com/mikefarah/yq
+    Example command to install yq:
+    sudo wget https://github.com/mikefarah/yq/releases/download/v4.9.8/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq'
+    if ! command -v jq &> /dev/null; then
         echo -e "JQ: https://stedolan.github.io/jq/download/"
     fi
-    echo -e
-    echo -e "You may also try installing YQ with PIP: pip install yq"
-    echo -e ""
-    echo
+    err 'You may also try installing YQ with PIP: pip install yq'
     exit 1
 else
   VERSION_YQ=$(yq --version | cut -d' ' -f3 | tr -d '"')
+
+  if [ "$VERSION_YQ" == "version" ]; then
+    VERSION_YQ=$(yq --version | cut -d' ' -f4 | tr -d '"')
+  fi
+
   echo "Using yq version $VERSION_YQ"
 
+  if [ "$(versionNumber $VERSION_YQ)" -lt "$(versionNumber '4.0.0')" ]; then
 
-  if [ $(versionNumber $VERSION_YQ) -lt $(versionNumber '4.0.0') ]; then
-      echo -e
-      echo -e "\e[91mERROR: \e[37m yq in 4+ version is required to install Dynatrace function. Please refer to following links for installation instructions:"
-      echo -e "YQ: https://github.com/mikefarah/yq"
-      echo -e
+      err 'yq in 4+ version is required to install Dynatrace function. Please refer to following links for installation instructions:
+      YQ: https://github.com/mikefarah/yq'
       exit 1
   fi
 fi
 
+if ! command -v gcloud &>/dev/null; then
 
-if ! command -v gcloud &> /dev/null
-then
-
-    echo -e "\e[91mERROR: \e[37mGoogle Cloud CLI is required to install Dynatrace function. Go to following link in your browser and download latest version of Cloud SDK:"
-    echo -e
-    echo -e "https://cloud.google.com/sdk/docs#install_the_latest_cloud_tools_version_cloudsdk_current_version"
-    echo -e
-    echo
-    exit
+  echo -e "\e[91mERROR: \e[37mGoogle Cloud CLI is required to install Dynatrace function. Go to following link in your browser and download latest version of Cloud SDK:"
+  echo -e
+  echo -e "https://cloud.google.com/sdk/docs#install_the_latest_cloud_tools_version_cloudsdk_current_version"
+  echo -e
+  echo
+  exit
 fi
 
-if ! command -v unzip &> /dev/null
-then
-    echo -e "\e[91mERROR: \e[37munzip is required to install Dynatrace function"
-    echo
-    exit
+if ! command -v unzip &>/dev/null; then
+  echo -e "\e[91mERROR: \e[37munzip is required to install Dynatrace function"
```

Comment:
Minor: It might be printed with err function

Reply:
the same